plonetheme/barceloneta/theme/less/loginform.plone.less

```diff
@@ -1,7 +1,8 @@
 //*// LOGIN FORM //*//
 
 body.template-login_form #content-core,
-body.template-logged_out #content-core {
+body.template-logged_out #content-core ,
+body.template-login #content-core {
```

Comment:
why introduce a new class instead of using the old one in plone.login templates?